### PR TITLE
Changes to target files to catch up with betaflight

### PR DESCRIPTION
**Pull request by martinbudden**
Changes in target files to reflect changes in baseflight.

### CODE REVIEW

**Comment by digitalentity** [comment]
Please don't merge this yet, I'll check that we didn't break anything.

File: src/main/target/ALIENFLIGHTF3/target.mk [hunk 1]
```diff
@@ -1,14 +1,16 @@
 F3_TARGETS  += $(TARGET)
-FEATURES    = VCP HIGHEND
```
Comment (digitalentity) [inline src/main/target/ALIENFLIGHTF3/target.mk:2]: Is this correct?

File: src/main/target/STM32F3DISCOVERY/target.mk [hunk 2]
```diff
@@ -1,5 +1,5 @@
 F3_TARGETS  += $(TARGET)
-FEATURES    = VCP SDCARD HIGHEND
```
Comment (digitalentity) [inline src/main/target/STM32F3DISCOVERY/target.mk:2]: Don't we use HIGHEND feature anymore?

Comment (martinbudden) [reply src/main/target/STM32F3DISCOVERY/target.mk:2]: HIGHEND is set in the `makefile` for all F3 and F4 targets.

**Pull request merged**